dockerfiles/bookworm/pip.conf

```diff
@@ -0,0 +1,2 @@
+[global]
```

Comment:
Can we create this config file from dockerfile through `echo > ..` ?

Reply:
I followed the approach that is used in the SONiC buildimage repository.

Reply:
@andriy-kokhan Done.